app/api/sponsors.py

```diff
@@ -109,7 +111,8 @@ class SponsorDetail(ResourceDetail):
     """
     Sponsor detail by id
     """
-    decorators = (jwt_required,)
+    decorators = (api.has_permission('is_coorganizer', methods="PATCH,DELETE",
```

Comment:
@poush Sorry. I was using a super-admin account, that's why it was working maybe :sweat: . Resolved here and in microlocations too